slack/README.md

```diff
@@ -0,0 +1,125 @@
+# Slack Guidelines
+
+Slack is the main communication platform for Kubeflow outside of our mailing
+lists. It’s important that conversation stays on topic in each channel, and that
+everyone abides by the [Code of Conduct](https://github.com/kubeflow/community/blob/master/CODE_OF_CONDUCT.md). Community
+members should all expect to have a positive experience.
+
+Chat is searchable and public. Do not make comments that you would not say on a
+video recording or in another public space. Please be courteous to others.
+
+`@here` and `@channel` should be used rarely. Members will receive notifications
+from these commands and we are a global project - please be kind. Note: `@all`
+is only to be used by admins.
+
+To join the [Kubeflow Slack](https://kubeflow.slack.com), click this [Invitation to Kubeflow workspace](https://join.slack.com/t/kubeflow/shared_invite/zt-cpr020z4-PfcAue_2nw67~iIDy7maAQ)
+
+## Code of Conduct
+
+The Kubeflow [Code of Conduct](https://github.com/kubeflow/community/blob/master/CODE_OF_CONDUCT.md) applies throughout the
+project, and includes all communication mediums.
+
+## Admins
+
+(by Slack ID)\
+@Igor Mameshin\
+@Jeremy Lewi
+
+To connect: please reach out in the #admins channel, mention one of us in the specific channel where you have a question, or DM (Direct Message) one of us privately.
+
+### Admin expectations and guidelines
+
+- Adhere to Code of Conduct
+- Take care of spam as soon as possible, which may mean taking action by making
+  members inactive
+- Moderating and fostering a safe environment for conversations
+- Bring Code of Conduct issues to the Steering Committee
+- Create relevant channels and list Code of Conduct in new channel welcome
+  message
+- Help troubleshoot Slack issues
+- Review bot, token, and webhook requests
+- Be helpful!
+
+## Creating channels
+
+Please submit a PR to [Slack Channels](./slack_channels.md) with the name and description for a new channel.
+You can also contact #admins group with your request to create a new channel.
+
+Channels are dedicated to working groups, sub-projects, community topics,
+and related programs/projects.
+
+Channels are not:
+
+- company specific; e.g. a channel named for a cloud provider must be used for
+  conversation about Kubeflow-related topics on that cloud, and not proprietary
+  information of the provider.
+- private unless there is an exception: code of conduct matters, mentoring,
+  security/vulnerabilities, or steering committee.
+
+All channels need a documented purpose. Use this space to welcome the targeted
+community: promote your meetings, post agendas, etc.
+
+We may make special accommodations where necessary.
+
+## Escalating and/or reporting a problem
+
+Join the #admins channel or contact one of the admins via DM directly and describe the situation. If the issue can be
+documented, please take a screenshot to include in your message.
+
+## Bots, tokens, and webhooks
```

Comment:
I think we've reached the quota limits w.r.t bots

Reply:
Sounds good, we will have to deny most requests for Slack bots.